_includes/chapter2.html

```diff
@@ -7,7 +7,7 @@ <h1><a id="chapter2">Verifiable deployments</a></h1>
 </p>
 <h2>General testing concepts</h2>
 <p>
-    Testing is a nebulous term with often ill-defined subcategories. We will not attempt to provide authoritative definitions of testing categories here. Our goal is to offer a very high level description of common testing practices, and highlight those that can be performed during the deployment process.
+    Testing is a nebulous term with often ill-defined subcategories. We won't attempt to provide authoritative definitions of testing categories here. Our goal is to offer a high level description of common testing practices, and highlight which can be performed during the deployment process.
 </p>
 <h2>What don't we test during deployments?</h2>
```

Comment:
@wordlee I'm not sure if this explicitly answers the question "what don't we test?". Not sure if the word "don't" is missing or if I'm misunderstanding the answer.

Reply:
I think it covers it by saying that these tests are run during a different stage.